src/Datagrid/Datagrid.php

```diff
@@ -291,4 +291,25 @@ public function getForm()
 
         return $this->form;
     }
+
+    /**
+     * Returns the default filter value if the user has not set it.
+     */
+    protected function resolveFilterValue(?array $data, string $name, FilterInterface $filter): ?array
```

Comment:
Why this needs to be protected?

Reply:
I think it should not be public. Do you suggest to make it private?

Reply:
@gam6itko Yes, private is better